test/polar_plot_cli.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:io';
+import 'dart:async';
+import 'package:sail_routing_dart/polar_plotting/polar_plot.dart';
+import 'package:sail_routing_dart/polar_algs/polar_router.dart';
+import 'package:sail_routing_dart/route_writer.dart';
+
+Future main (List<String> arguments) async {
+    exitCode = 0;
+    print("hey look at my arguments! ");
+    for (final arg in arguments) {
+        print(arg);
+    }
+    var plot = PolarPlot();
```

Comment:
Would be nice to see documentation on what this is trying to do (or at least the overall process). Example: Construct a plot. Passing in some points to the routing section. Plot those points. Visually show what was done.